include/myst/syscall.h

```diff
@@ -407,4 +407,21 @@ typedef struct myst_syscall_pair
 
 const myst_syscall_pair_t* myst_syscall_pairs(void);
 
+#define SYSCALL_GROUP_MAX_SIZE 128
+
+/* Stores the syscall group name, corresponding syscalls, and number of syscalls
+ */
+typedef struct myst_syscall_group
+{
+    const char* name;
+    const size_t group_size;
+    const int syscalls[SYSCALL_GROUP_MAX_SIZE];
+} myst_syscall_group_t;
+
+const myst_syscall_group_t* myst_syscall_groups(void);
```

Comment:
There is an informal naming convention for interface functions of kernel subsytems - `myst_<subsytem-name>_<function-name>`.
```suggestion
const myst_syscall_group_t* myst_strace_syscall_groups(void);
```

Reply:
I went by the convention for the pairs structure defined in this file for this one - the structure should be a general kernal differentiation rather than something just for strace right? We also define it in syscall.c and not strace.h

Reply:
Even `_pairs` is used just by the tracing subsystem.